shpc/main/modules/base.py

```diff
@@ -517,11 +512,14 @@ def _set_default_version(self, version_file, tag):
         template = self._load_template("default_version")
         utils.write_file(version_file, template.render(version=tag))
 
-    def write_version_file(self, uri, latest_tag_installed=None):
+    def write_version_file(self, version_dir, latest_tag_installed=None):
         """
         Create the .version file, if there is a template for it.
         """
-        version_dir = os.path.join(self.settings.module_base, uri)
```

Comment:
Why did you move this inside the function? (the derivation of version_dir)?

Reply:
This function is now called for directories under `$module_base` _and_ `$symlink_base`. It can't expect everything to be under `$module_base`.

Reply:
Yep that makes sense! Thank you for the explanation.